parser/parser.y

```diff
@@ -3667,6 +3667,17 @@ DatabaseOption:
 			UintValue: placementOptions.UintValue,
 		}
 	}
+|	"SET" "TIFLASH" "REPLICA" LengthNum LocationLabelList
```

Comment:
If we do it here, we need to support `"SET" "TIFLASH" "REPLICA" LengthNum LocationLabelList "SET" "TIFLASH" "REPLICA" LengthNum LocationLabelList`. 
Is this expected? Add relevant tests if expected.

Reply:
Could we add some tests to `parser_test.go`?